examples/demo13.py

```diff
@@ -93,11 +93,11 @@ def main(argv):
     logger.debug('Read in WFIRST imaging filters.')
 
     logger.info('Reading from a parametric COSMOS catalog.')
-    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<23.5 from COSMOS.
+    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<25.2 from COSMOS.
```

Comment:
No, I did not update the example catalog (no reason to - COSMOSCatalog has multiple possible samples it can take, and there's no reason why the example has to be updated just because the default COSMOSCatalog choice changed).  I thought the changes in these comments meant you had updated it yourself and then not put the update in the repo.  As I said, if you didn't do that, then your changes to the comments should be reverted.